.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: ruby 
 ruby:
      - 2.6.3 
-script: bundle exec rspec
\ No newline at end of file
+script: 
+  - bundle exec rspec
+  - bundle exec standardrb
```

Comment:
Just an FYI, it might be possible to run `rspec` and `standardrb` as two separate jobs. That might give you more targetted feedback if one of the tasks fails.

Reply:
Here are the docs for that if you are interested. https://docs.travis-ci.com/user/build-stages/

Reply:
Incorporated. Thanks.